### ISSUE TITLE
A member of 'a team' can't create release issue

### ISSUE THREAD

**Issue by jpopelka** [user-cont/release-bot#150]
For example in https://github.com/packit-service/ogr/issues/15 @lachmanfrantisek was ignored because
```
WARNING Author association 'CONTRIBUTOR' not in ['MEMBER', 'OWNER', 'COLLABORATOR']
```
even he was in part of 'The packit team' whose members should be admins in the ogr project.

Once we added him as collaborator, the issue was processed.

**Comment by TomasTomecek** [comment]
github roles are very clear and easy to use... FML

**Comment by jpopelka** [comment]
Ok, so I'm confused now since my https://github.com/user-cont/conu/issues/353 was processed even I'm not explicitly added as collaborator in https://github.com/user-cont/conu/settings/collaboration

**Comment by TomasTomecek** [comment]
could it be just a race condition that github did not update the roles in their system in time and their API returned outdated info?

**Comment by jpopelka** [comment]
I don't think so, the 'The packit team' was added some time ago and also Franta has been a member for some time.

**Comment by TomasTomecek** [comment]
oki, we should investigate and resolve this then
